man/man8/facter.8

```diff
@@ -45,14 +78,49 @@ hardwaremodel => x86_64
 .IP "" 0
 .
 .P
-Display a single fact:
+Display a single structured fact:
 .
 .IP "" 4
 .
 .nf
 
-$ facter kernel
-Linux
+$ facter processors
+{
+  count => 2,
+  isa => "x86_64",
+  models => [
+    "Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz",
+    "Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz"
+  ],
+  physicalcount => 2
+}
+.
+.fi
+.
+.IP "" 0
+.
+.P
+Display a single fact nested within a structured fact:
+.
+.IP "" 4
+.
+.nf
+$ facter processors.isa
+x86_64
+.
+.fi
+.
+.IP "" 0
+.
+.P
+Display a single legacy fact. Note that non-structured facts existing in previous versions of Facter are still available,
+but are not displayed by default due to redundancy with newer structured facts:
```

Comment:
@MikaelSmith @peterhuene cool, added a bit of wording to the example explain legacy facts. How's this sound?